Web3Swift/TransactionBytes/EthDirectTransactionBytes.swift

```diff
@@ -0,0 +1,125 @@
+/**
+Copyright 2018 Timofey Solonin
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import Foundation
+
+//Bytes of a signed transaction to ethereum address
+public final class EthDirectTransactionBytes: BytesScalar {
```

Comment:
@biboran Seven ctor parameters seems to be too much. What do you think about splitting the class into smaller ones?